html/components/button.stories.js

```diff
@@ -23,11 +23,24 @@ always use a \`<button>\` element.
 must include a \`title=””\` attribute.
 - If a Button only includes an Icon with no text,
 alternative text must be provided.
-- If the Button has a spinner setup, the \`aria-label\`
-will be set to 'Loading' when the button is
-clicked unless you supply a custom
-string value to the \`data-sprk-spinner-aria-label\`
-attribute.
+
+##### Spinner State
+- Spinner state requires additional engineering.
+These stories only represent a spinner button after javascript intervention.
+- Add these attributes on buttons that will have a spinning state:
+  - \`class="sprk-c-Button"\` as well as any class modifiers.
+  - \`data-sprk-spinner="click"\`
+  - \`aria-live="polite"\`
+  - Other attributes seen in the story is
+  added after calling the \`setSpinning\` function.
+- Use \`setSpinning\` and \`cancelSpinning\` functions
+to stop start or stop a button from spinning.
+These functions adds other necessary classes,
+attributes and accessibility considerations.
+- The following data attributes override accessibility defaults.
+  - \`data-sprk-spinner-role=""\` - Defaults to "progressbar".
+  - \`data-sprk-spinner-aria-valuetext=""\` - Defaults to "Loading"
+  - \`data-sprk-spinner-aria-label=""\` - Defaults to "Loading"
```

Comment:
@afebbraro Any idea how hard it would be for us to add a prop table to the HTML storybook like we have for angular and react?

Reply:
I don't think its offered in the version we have